core/src/com/unciv/models/ruleset/unique/IHasUniques.kt

```diff
@@ -17,8 +17,12 @@ interface IHasUniques : INamed {
     val uniqueMap: Map<String, List<Unique>>
 
     fun uniqueObjectsProvider(): List<Unique> {
-        if (uniques.isEmpty()) return emptyList()
-        return uniques.map { Unique(it, getUniqueTarget(), name) }
+        var uniqueObjectList: List<Unique>
+        if (uniques.isEmpty())
+            uniqueObjectList = emptyList()
+        else
+            uniqueObjectList = uniques.map { Unique(it, getUniqueTarget(), name) }
+        return uniqueObjectList + legacyRequiredTechsAsUniques().toList()
```

Comment:
I see no reason to do this, and a bunch of ways this could mess with us later
If we need to handle tech uniques differently, this is not the way